util/util.py

```diff
@@ -9,6 +9,68 @@
 import collections
 import math
 import torch.nn as nn
+import torch.nn.functional as F
+
+
+class OptimizerMask:
```

Comment:
This class seems not to be used anywhere. Wonder whether the adoption of this class has been forgotten somewhere else.